articles/postgresql/concepts-server-logs.md

```diff
@@ -17,22 +17,22 @@ Azure Database for PostgreSQL generates query and error logs. However, access to
 You can list and download Azure PostgreSQL server error logs using the Azure portal, [Azure CLI](howto-configure-server-logs-using-cli.md), and Azure REST APIs.
 
 ## Log retention
-You can set the retention period for system logs using the **log\_retention\_period** parameter associated with your server. The unit for this parameter is days. The default value is three days. The maximum value is 7 days. Your server must have enough allocated storage to contain the retained log files.
+You can set the retention period for system logs using the **log\_retention\_period** parameter associated with your server. The unit for this parameter is days. The default value is 3 days. The maximum value is 7 days. Your server must have enough allocated storage to contain the retained log files.
```

Comment:
The next sentence uses the numeric value 7, so for consistency, change "three" to "3"

Reply:
We have certain review tools that check this and suggest which ones to spell out. I think the numbers are fine here, but typically small numbers are spelled out.

Reply:
OK. Could have gone either way, I just wanted to be consistent.